context/src/junit/validator.rs

```diff
@@ -607,12 +607,12 @@ pub enum JunitTestCaseValidationIssueSubOptimal {
         TIMESTAMP_STALE_HOURS
     )]
     TestCaseStaleTimestamp(DateTime<FixedOffset>),
+    #[error("test case id is not a valid uuidv5")]
+    TestCaseInvalidId(String),
```

Comment:
Going forward, this won't be a real issue. I want to remove it before release, but will leave it as suboptimal for tracking.

Reply:
Do we have any times where the id is provided to us in the junit to begin with? I asusme we only generate this when unset for a reason...

Reply:
Yes, there's a few test runners that set an id field.